sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -93,6 +93,19 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
     NOTE: If you are using WSL, do not run these commands from the Windows system drive (e.g. `/mnt/c/`).
 
+    ### Certificates - Important to know
+
+    For sample purposes, Embedded C SDK samples leverage certificates obtained when executing either `New-ArduinoZipLibrary.ps1` or `generate_arduino_zip_library.sh`. 
+    When creating solutions based on the Embedded C SDK you should consider :
```

Comment:
```suggestion
    When creating solutions based on the Embedded C SDK that use certificates, you should consider the following:
```

Reply:
I don't think this is giving the right idea.
There are two types of certs every application (including samples) can work with.
- Root CAs used to validate the server cert. These shall always be employed by applications. We do not recommend people to skip server cert validation.
- Client x509 certs used for device authentication; these are optional (although also recommended) since customers can use SAS tokens.

Reply:
applied the suggestion from your last comment